README.md

```diff
@@ -120,6 +120,42 @@ Call the `feather.replace()` method.
 
 All elements that have a `data-feather` attribute will be replaced with SVG markup corresponding to their `data-feather` attribute value. See the [API Reference](#api-reference) for more information about `feather.replace()`.
 
+#### 5. SVG Sprite
```

Comment:
I feel like this heading should be on the same level as "Client-side" and "Node." The idea was that each subheading of "Usage" is a different way to use Feather. I'm planning to add sections like "React" and "Sketch" soon.

Reply:
I'm already using the sprite with React, but I was thinking that using the icons.json to render a React Component is even better.

Reply:
Yeah, totally. I was just suggesting that the `SVG Sprite` section should be on the same hierarchical level as `Client-side` and `Node`.